docs/md/sink.md

```diff
@@ -0,0 +1,132 @@
+# Sink
+The workload `Sink` models the terminal stage of a processing pipeline. In a 
+typical workload configuration, a `Sink` can be used to write workflow outputs
+to a desired location in the cloud.
+
+## User Guide
+You can configure the type of `Sink` used in your workload by changing the
+`sink` attribute of your workload request.
+
+### Terra Workspace Sink
+You can write workflow outputs to a Terra Workspace using the `Terra Workspace`
+sink. A typical `Terra Workspace` sink configuration in the workload request
+looks like:
+```json
+{
+  "name": "Terra Workspace",
+  "workspace": "{workspace-namespace}/{workspace-name}",
+  "entityType": "{name}",
```

Comment:
Having `{name}` as the value here is a bit confusing to me given the `name` key above, would qualify.

Reply:
good catch